apps/style/applab/style.scss

```diff
@@ -41,6 +41,9 @@ div#visualization.with_padding {
   html[dir='rtl'] div#visualizationResizeBar {
     right: $width;
   }
+  #visualizationResizeIcon {
```

Comment:
to ensure that the applab specific styles override the common ones, this file should use `div#visualizationResizeIcon` (both here and the rtl version below)